cluster/manifests/coredns/service-coredns.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
 spec:
   selector:
     application: coredns
+    instance: cluster-dns
```

Comment:
Is there a reason for not just pointing this service at all coredns pods whether they are daemonset of deployment pods? This would also avoid adding an extra service in #1612